### PR TITLE
EG-4171: final re-arrangement of upgrade files

### PR DESCRIPTION
**Pull request by AMcFarlandCampbellR3**

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.6/upgrading-index.md [hunk 1]
```diff
@@ -7,13 +7,19 @@ menu:
     parent: corda-enterprise-4-6-upgrading-and-tools
 tags:
 - upgrading
-title: Updating a CorDapp or Node
+title: Updating a CorDapp or node
 weight: 1
 ---
 
-# Updating to a new version
+# Updating a CorDapp or node
```
Comment (taylorharvey-r3) [inline content/en/docs/corda-enterprise/4.6/upgrading-index.md:14]:
```suggestion
# Upgrading a CorDapp or node
```
Should this be upgrading? It's the only place I've seen "Updating".

**Review by taylorharvey-r3 — APPROVED** [review]
One question but overall LGTM.

File: content/en/docs/corda-enterprise/4.6/upgrading-index.md (hunk 1 above)
Comment (AMcFarlandCampbellR3) [reply content/en/docs/corda-enterprise/4.6/upgrading-index.md]: I've made this change. Resolving conversation.

**Review by taylorharvey-r3 — APPROVED** [review]
LGTM

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Pull request merged**